src/display/api.js

```diff
@@ -169,6 +169,9 @@ function setPDFNetworkStreamFactory(pdfNetworkStreamFactory) {
  *   of PDF files. When enabled, and if the server supports partial content
  *   requests, then the PDF will be fetched in chunks.
  *   The default value is `false`.
```

Comment:
It looks like the commit message of this commit is copy-pasted from the first one ;-)